src/main/java/Jeoe/Jeoe.java

```diff
@@ -0,0 +1,89 @@
+package Jeoe;
```

Comment:
Shouldn't the package names be all lowercase? I noticed this in other sub-packages too.